### DISCUSSION TITLE
Coolify should not need to root the system

### DISCUSSION THREAD

**Discussion by Bilge** [coollabsio/coolify#4165] [Improvement Requests]
Installing an SSH key under `root` is neither secure nor should it be necessary. It is currently entirely opaque to me why this is supposedly needed. As far as I can tell, the only thing Coolify needs to do is communicate with the Docker daemon, which does not require root access.

**Comment by f1yn** [comment]
I've been trying Coolify recently for production use, and I've reached this same conclusion. I've been using `podman` (which is rootless) before Coolify, and I didn't realize how good I've had it.

This choice alone is making me consider having Coolify sandboxed to a VM, or just going back to podman for better security.
